src/types/fhir-api-types.ts

```diff
@@ -54,14 +54,14 @@ export type FhirAPIEndpoints = {
     Response: WithIdDefined<FhirResourcesByName[Name]>;
   };
 } & {
-  // PUT /<resource>/:id
-  [Name in keyof FhirResourcesByName as `PUT /v1/fhir/dstu3/${Name}/:id`]: {
+  // PUT /<resource>/:resourceId
+  [Name in keyof FhirResourcesByName as `PUT /v1/fhir/dstu3/${Name}/:resourceId`]: {
     Request: WithIdDefined<FhirResourcesByName[Name]>;
     Response: WithIdDefined<FhirResourcesByName[Name]>;
   };
 } & {
-  // GET /<resource>/:id
-  [Name in keyof FhirResourcesByName as `GET /v1/fhir/dstu3/${Name}/:id`]: {
+  // GET /<resource>/:resourceId
```

Comment:
Can we leave this one as-is? I don't think we need to change it.

Reply:
My thought here is that we'd change both places for consistency, but not a hard opinion. Updating.